### ISSUE TITLE
Matlab example

### ISSUE THREAD

**Issue by bkmgit** [hpcg-benchmark/hpcg#61]
The current Matlab/Octave example code ([here](https://github.com/hpcg-benchmark/hpcg/blob/master/tools/hpcg.m)) does not use a multigrid preconditioner and requires generation of the matrix by the C code. While this is useful for validating the C code, it can be useful to also generate the matrix directly in Matlab/Octave. Can contribute such a code if it would be helpful.

**Comment by luszczek** [comment]
Yes, please feel free to submit a pull request with your contribution.
